docs/user-guide/database-configuration.md

```diff
@@ -58,7 +58,7 @@ The following variables are customizable for the managed PostgreSQL service
 
 | Name                                          | Description                                   | Default                            |
 | --------------------------------------------- | --------------------------------------------- | ---------------------------------- |
-| postgres_image                                | Path of the image to pull                     | postgres:12                        |
+| postgres_image                                | Path of the image to pull                     | postgres:15                        |
```

Comment:
Should be `quay.io/sclorg/postgresql-15-c9s:latest`

Reply:
Good catch, I've updated it.  Thanks!